pkg/route/api/validation/validation.go

```diff
@@ -300,9 +301,14 @@ func validateCertificatePEM(certPEM string, options *x509.VerifyOptions) (*x509.
 	}
 
 	if options != nil {
+		// Ensure we don't report errors for expired certs or if
+		// the validity is in the future.
+		options.CurrentTime = cert.NotAfter.AddDate(0, 0, -1)
```

Comment:
I'm wondering if we can just validate the well-formedness of the CA bundle and skip the validation.